jetstream/cli.py

```diff
@@ -362,8 +347,12 @@ def ensure_enrollments(
 )
 @click.option("--log_table_id", "--log-table-id", default="logs", help="Table to write logs to")
 @click.option("--log_to_bigquery", "--log-to-bigquery", is_flag=True, default=False)
-def cli(log_project_id, log_dataset_id, log_table_id, log_to_bigquery):
-    setup_logger(log_project_id, log_dataset_id, log_table_id, log_to_bigquery)
+@click.pass_context
+def cli(ctx, log_project_id, log_dataset_id, log_table_id, log_to_bigquery):
```

Comment:
Alternatively, we could also hardcode this. We don't actually pass it into the workflow config when running via Argo. So, it always uses the default config then.